src/__tests__/__snapshots__/rules.test.ts.snap

```diff
@@ -91,6 +91,7 @@ Object {
     ],
     "rules": Object {
       "jest/no-alias-methods": "warn",
+      "jest/prefer-to-be": "error",
       "jest/prefer-to-be-null": "error",
       "jest/prefer-to-be-undefined": "error",
```

Comment:
these should be removed first (or at the same time)

Reply:
Yeah I wanted to make the PR without setting it up to conflict when #918 is merged in